docs/source/challenge_creation.md

```diff
@@ -141,6 +141,68 @@ The final step is to create a challenge host team for the challenge on EvalAI. A
 
 If you have issues in creating a challenge on EvalAI, please feel free to create an issue on our [Github Issues Page](https://github.com/Cloud-CV/EvalAI/issues/new).
 
+#### How to approve a challenge?
```

Comment:
Ideally, this should be a new markdown file instead of overwhelming the challenge creation markdown. I think that's what we converged on.